### ISSUE TITLE
Enable export to CSV/XLS

### ISSUE THREAD

**Issue by jackiehluo** [coreatcu/contacts#4]
Would be nice to be able to download a CSV/XLS file of the database (or, better yet, specific query results from the database).

**Comment by joshuazweig** [comment]
I played around with download as a csv for a project I worked on last year
Not to difficult and happy to take a crack at it. 
Making the cvs is pretty easy, the hard part is coming up with an easy way to serve the file

**Comment by jackiehluo** [comment]
@joshuazweig Yeah, it should be super, super easy! There are lots of Rails gems that do it, and I wanted to make sure that I remembered to do it. But assign yourself an issue/send a pull request anytime! :+1:
